server/src/util.js

```diff
@@ -0,0 +1,58 @@
+const WebSocket = require('ws');
+const shortid = require('shortid');
+const nodeMailer = require('nodemailer');
+
+const baseUrl = process.env.baseUrl || `localhost:3000`
+const transporter = nodeMailer.createTransport({
+  host: 'smtp.gmail.com',
+  port: 465,
+  secure: true,
+  auth: {
+    user: 'lucent.emdr@gmail.com',
+    pass: 'lucent316!'
```

Comment:
let's remove this so it's not committed to the repo. You can save in a `.gitignore`d file for configuration.json on the root that you can require

Reply:
i saw the commit. cool cool